File: clients/dotnet/models/PingResultModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// Ping Result Model
    /// </summary>
    public class PingResultModel
    {
        /// <summary>
        /// Version number
        /// </summary>
        public String version { get; set; }

        /// <summary>
        /// Returns true if you provided authentication for this API call; false if you did not.
        /// </summary>
        public Boolean? authenticated { get; set; }

        /// <summary>
        /// Returns the type of authentication you provided, if authenticated
        /// </summary>
        public AuthenticationTypeId? authenticationType { get; set; }

        /// <summary>
        /// The username of the currently authenticated user, if any.
        /// </summary>
        public String authenticatedUserName { get; set; }


    }
}

File: clients/php/models/PingResultModel.php
<?php 
namespace Avalara;
/*
 * AvaTax API Client Library
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 *
 * @category   AvaTax client libraries
 * @package    Avalara.AvaTaxClient
 * @author     Ted Spence <ted.spence@avalara.com>
 * @author     Bob Maidens <bob.maidens@avalara.com>
 * @copyright  2004-2016 Avalara, Inc.
 * @license    https://www.apache.org/licenses/LICENSE-2.0
 * @version    2.16.12-30
 * @link       https://github.com/avadev/AvaTaxClientLibrary
 */


/**
 * Ping Result Model
 */
final class PingResultModel
{
    /**
     * @var String Version number
     */
    public $version;

    /**
     * @var Boolean? Returns true if you provided authentication for this API call; false if you did not.
     */
    public $authenticated;

    /**
     * @var AuthenticationTypeId? Returns the type of authentication you provided, if authenticated
     */
    public $authenticationType;

    /**
     * @var String The username of the currently authenticated user, if any.
     */
    public $authenticatedUserName;

}
